### PR TITLE
Player 1788b

### PR DESCRIPTION
**Pull request by aeng7**
-workaround of an IMA issue where the ad starts paused on Safari 11 with our muted autoplay flow
-modified the setVolume function to set the volume as long as the IMA Ads Manager exists, and will toss the previous saved volume when doing so. This allows us to use this function to prime future ads for unmuted autoplay when called on a user click

### CODE REVIEW

**Comment by pilievOoyala** [comment]
LGTM

File: js/google_ima.js [hunk 1]
```diff
@@ -668,8 +668,11 @@ require("../html5-common/js/utils/utils.js");
 
       this.setVolume = function(volume)
       {
-        if (_IMAAdsManager && _linearAdIsPlaying)
+        if (_IMAAdsManager)
         {
+          if (typeof this.savedVolume === "number") {
```
Comment (jpfranco) [inline js/google_ima.js]: Should this be `!==`?

Comment (aeng7) [reply js/google_ima.js]: Nope, because I'm getting rid of the savedVolume at this point. I technically don't need this type check though. I'll remove it.

Comment (jpfranco) [reply js/google_ima.js]: Oh ok I see

**Pull request merged**